doc/index.rst

```diff
@@ -0,0 +1,59 @@
+yaml2sbml
+=========
+
+.. image:: https://github.com/yaml2sbml-dev/yaml2sbml/workflows/CI/badge.svg
+   :target: https://github.com/yaml2sbml-dev/yaml2sbml/actions
+   :alt: Build status
+.. image:: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml
+   :alt: Code coverage
+.. image:: https://readthedocs.org/projects/yaml2sbml/badge/?version=latest
+   :target: https://yaml2sbml.readthedocs.io/en/latest/?badge=latest
+   :alt: Documentation Status
+.. image:: https://app.codacy.com/project/badge/Grade/dc25c9a84ba54710bbb23a6e08ab5d22
+   :target: https://www.codacy.com/manual/martamatos/yaml2sbml/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=yaml2sbml-dev/yaml2sbml&amp;utm_campaign=Badge_Grade
```

Comment:
Old location of `martamatos/yam2sbml`, and not sure if some/all of the `utm` parameters are necessary

Reply:
nice catch!